### PR TITLE
Update free-programming-books.md

### PR DESCRIPTION
**Pull request by pichonz**
Updated pandas section, added tornado tutorial notebooks, added tensorflow tutorial notebooks.

### CODE REVIEW

**Comment by pichonz** [comment]
What went wrong, let me know and I'll try to make any corrections on my end.

File: free-programming-books.md [hunk 1]
```diff
@@ -1850,13 +1852,17 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
...
 
 
 #### Pyramid
 
 * [Pyramid for humans](http://docs.pylonsproject.org/projects/pyramid-tutorials/en/latest/humans/index.html)
 
+#### Tornado
+
+* [Learn Web Programming](https://bitbucket.org/hrojas/learn-web-programming)
+
```
Comment (eshellman) [inline free-programming-books.md]: too many spaces

Comment (pichonz) [inline free-programming-books.md]: thanks, anything left to do on my end?

Comment (eshellman) [inline free-programming-books.md]: Yes, remove the extra spaces and add a commit. Travis CI will check your work.

Comment (pichonz) [inline free-programming-books.md]: I tried to remove any extra spaces but I didn't see any. Maybe I missing something real obvious. I tried to commit again.

**Comment by eshellman** [comment]
needs extra line before "#### Tornado".  I'm just starting to understand this myself!

**Comment by pichonz** [comment]
I think I finally got it.

**Comment by vhf** [comment]
Thanks!

**Pull request merged**